<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="3" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20429" started-at="2018-01-04T06:16:31Z" finished-at="2018-01-04T06:16:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20429" started-at="2018-01-04T06:16:31Z" finished-at="2018-01-04T06:16:51Z">
      <class name="Sam.com.actiance.Teams.FinalTestCase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:Sam.com.actiance.Teams.FinalTestCase@5e3a8624]" name="setUp" is-config="true" duration-ms="5297" started-at="2018-01-04T11:46:31Z" finished-at="2018-01-04T11:46:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="NwayChat()[pri:3, instance:Sam.com.actiance.Teams.FinalTestCase@5e3a8624]" name="NwayChat" duration-ms="15091" started-at="2018-01-04T11:46:36Z" finished-at="2018-01-04T11:46:51Z">
          <exception class="com.microsoft.sqlserver.jdbc.SQLServerException">
            <message>
              <![CDATA[The TCP/IP connection to the host 192.168.117.113, port 1433 has failed. Error: "connect timed out. Verify the connection properties. Make sure that an instance of SQL Server is running on the host and accepting TCP/IP connections at the port. Make sure that TCP connections to the port are not blocked by a firewall.".]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.microsoft.sqlserver.jdbc.SQLServerException: The TCP/IP connection to the host 192.168.117.113, port 1433 has failed. Error: "connect timed out. Verify the connection properties. Make sure that an instance of SQL Server is running on the host and accepting TCP/IP connections at the port. Make sure that TCP connections to the port are not blocked by a firewall.".
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(SQLServerException.java:190)
	at com.microsoft.sqlserver.jdbc.SQLServerException.ConvertConnectExceptionToSQLServerException(SQLServerException.java:241)
	at com.microsoft.sqlserver.jdbc.SocketFinder.findSocket(IOBuffer.java:2243)
	at com.microsoft.sqlserver.jdbc.TDSChannel.open(IOBuffer.java:491)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(SQLServerConnection.java:1309)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.login(SQLServerConnection.java:991)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(SQLServerConnection.java:827)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(SQLServerDriver.java:1012)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at Sam.com.actiance.Teams.DatabaseDetails.VantageDBConnectivity(DatabaseDetails.java:36)
	at Sam.com.actiance.Teams.FinalTestCase.NwayChat(FinalTestCase.java:268)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- com.microsoft.sqlserver.jdbc.SQLServerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NwayChat -->
      </class> <!-- Sam.com.actiance.Teams.FinalTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
